builtin/testfile

```diff
@@ -0,0 +1 @@
+1,2,3,
```

Comment:
What is this file?

Reply:
The file seems to be uploaded incorrectly. I deleted it.